skopeo/test/dest-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Instead of passing the creds, could use such approach

Reply:
@hrishin which approach :sweat_smile: ?

Reply:
lol... sorry I mean like https://github.com/tektoncd/catalog/blob/v1beta1/buildah/tests/pre-apply-task-hook.sh @vinamra28

Reply:
aka using the local registry :angel:

Reply:
Okay, will update with that :slightly_smiling_face: